components/auto-complete/AutoComplete.tsx

```
@@ -132,6 +132,34 @@ const AutoComplete: React.ForwardRefRenderFunction<RefSelectProps, AutoCompleteP
+        const textarea = inputElement;
+        const cursorPos = textarea.selectionStart;
+
+        // Calculate which line the cursor is on
+        const textBeforeCursor = textarea.value.substring(0, cursorPos);
+        const currentLine = textBeforeCursor.split('\n').length - 1;
+
+        // Only adjust if cursor is not on first line
+        if (currentLine > 0) {
+          // Get line height from computed styles
+          const styles = window.getComputedStyle(textarea);
+          const lineHeight = parseInt(styles.lineHeight, 10) || 20;
```

ESLint

[error] Prefer `Number.parseInt` over `parseInt`.

(unicorn/prefer-number-properties)